iamctl/pkg/userStores/import.go

```diff
@@ -88,3 +95,21 @@ func importUserStore(userStoreId string, importFilePath string) error {
 	log.Println("User store imported successfully.")
 	return nil
 }
+
+func removeDeletedDeployedUserstores(localFiles []os.FileInfo, deployedUserstores []userStore) {
+
+	// Remove deployed user stores that do not exist locally.
+deployedResourcess:
```

Comment:
check formatting

Reply:
This is automatically reformatted to this, by the VS code lint extension.